app/namespace.go

```diff
@@ -57,6 +60,38 @@ var (
 		"us-east-1",
 		"us-west-2",
 	}
+	sinkNameFlag = &cli.StringFlag{
+		Name:     "sink-name",
+		Usage:    "sink name",
+		Required: true,
+	}
+	sinkEnabledFlag = &cli.BoolFlag{
+		Name:  "enabled",
+		Usage: "Whether the export sink is enabled",
+	}
+	sinkTypeFlag = &cli.StringFlag{
+		Name:     "type",
+		Usage:    "The type of the export sink",
+		Required: true,
+	}
+	sinkAssumedRoleFlag = &cli.StringFlag{
+		Name:     "assumed-role",
+		Usage:    "The assumed role for the sink",
+		Required: true,
+	}
+	sinkDestinationArnFlag = &cli.StringFlag{
+		Name:     "destination-arn",
+		Usage:    "The destination ARN for the sink",
```

Comment:
how is this going to work when we introduce GCP / other possible sinks here? I wonder if we need to have subcommands for different destination types so that users dont get over-whelmed by too many flags